custom_cards/custom_card_speedtest_shogun160/custom_card_speedtest_shogun160.yaml

```diff
@@ -0,0 +1,387 @@
+---
+custom_card_speedtest_shogun160:
+  template: "list_3_items"
+  triggers_update: "all"
+  show_name: false
+  show_label: false
+  variables:
```

Comment:
Wouldn't it be more user-friendly when there are at least defaults for the corresponding colors and max-values? This leaves users with only 3 variables to configure for a working card instead of 9.